model/src/main/kotlin/io/spine/validation/OptionNames.kt

```diff
@@ -98,3 +98,8 @@ public const val WHEN: String = "when"
  * The name of the `(goes)` option.
  */
 public const val GOES: String = "goes"
+
+/**
+ * The name of `(if_has_duplicates)` option.
+ */
+public const val IF_HAS_DUPLICATES: String = "if_has_duplicates"
```

Comment:
Can we get it via descriptors of options instead?

Reply:
@yevhenii-nadtochii as we have discussed here vocally just now, let's continue with the string values for now. 

Once we migrate the whole codebase to the current "new" validation approach, we'll be able to get rid of the string literals.